javascript/src/main/components/Statistics/FullCoursesForm.js

```diff
@@ -0,0 +1,46 @@
+import React, { useState } from "react";
+import { Form, Button, Spinner } from "react-bootstrap";
+import fetch from "isomorphic-unfetch";
+import DepartmentFormSelect from "main/components/Statistics/DepartmentFormSelect";
+import QuarterFormSelect from "main/components/Statistics/QuarterFormSelect";
+
+const FullCoursesForm = ({ setCourseJSON, fetchJSON, onSubmit = () => {} }) => {
+
+    const [startQuarter, setStartQuarter] = useState("20204");
+    const [endQuarter, setEndQuarter] = useState("20211");
+    const [department, setDepartment] = useState("CMPSC");
+    const [loading, setLoading] = useState(false);
+
+    const handleSubmit = (event) => {
+        event.preventDefault();
+        console.log("submit pressed");
```

Comment:
Seems to be unnecessary console.log statement for debugging. Consider removing

Reply:
Fixed!